web/components/templates/sessions/sessionsPage.tsx

```diff
@@ -438,9 +438,10 @@ const SessionsPage = (props: SessionsPageProps) => {
             activeColumns={activeColumns}
             setActiveColumns={setActiveColumns}
             rowLink={(row: TSessions) =>
-              `/sessions/${encodeURIComponent(
-                row.metadata.session_name
-              )}/${encodeURIComponent(row.metadata.session_id)}`
+              `/sessions/${
+                encodeURIComponent(row.metadata.session_name) ||
+                EMPTY_SESSION_NAME
```

Comment:
just to be a little more careful can we do this

```suggestion
                row.metadata.session_name ? encodeURIComponent(row.metadata.session_name) : EMPTY_SESSION_NAME
```

cause i just checked this and i'm not sure what we're storing it as
![image](https://github.com/user-attachments/assets/4c8dbd07-99b1-4dae-97d5-ee3d6b918895)